demo-apps/angular/src/styles.css

```diff
@@ -0,0 +1,4 @@
+/* You can add global styles to this file, and also import other style files */
+@import '@cdssnc/gcds-components/dist/gcds/gcds.css';
+@import '@cdssnc/gcds-utility/dist/gcds-utility.min.css';
```

Comment:
Can we add a comment for each of the imports here to make it easier to understand for users what they are?